.travis.yml

```diff
@@ -12,6 +12,15 @@ before_script:
 git:
   depth: 99999
 
+deploy:
+  provider: releases
+  api_key:
+    secure: $DEPLOY_TOKEN_SECURE
```

Comment:
Changed this to a variable so you can easily configure (and possibly easily change in future) it in future directly in Travis UI: https://travis-ci.org/Roave/BackwardCompatibilityCheck/settings

To create encrypted version of the API key, please follow: https://docs.travis-ci.com/user/encryption-keys/

```bash
travis encrypt -r Roave/BackwardCompatibilityCheck
```